core/server/master/src/test/java/alluxio/master/file/FileSystemMasterSyncMetadataMetricsTest.java

```diff
@@ -0,0 +1,705 @@
+import alluxio.master.metrics.MetricsMasterFactory;
+import alluxio.metrics.MetricsSystem;
+import alluxio.security.authentication.AuthenticatedClientUser;
+import alluxio.security.user.UserState;
+import alluxio.underfs.UfsFileStatus;
+import alluxio.underfs.UfsStatus;
+import alluxio.underfs.UfsStatusCache;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.underfs.UnderFileSystemConfiguration;
+import alluxio.underfs.local.LocalUnderFileSystem;
+import alluxio.util.ThreadFactoryUtils;
+import alluxio.util.executor.ExecutorServiceFactories;
+import alluxio.util.io.PathUtils;
+
+import com.codahale.metrics.Counter;
+import com.google.common.collect.ImmutableList;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({UnderFileSystem.Factory.class})
+public class FileSystemMasterSyncMetadataMetricsTest {
+  private static final AlluxioURI ROOT = new AlluxioURI("/");
+  private static final String TEST_DIR_PREFIX = "/dir";
+  private static final String TEST_FILE_PREFIX = "/file";
+  @Rule
+  public TemporaryFolder mTempDir = new TemporaryFolder();
+  private String mUfsUri;
+  private FlakyLocalUnderFileSystem mUfs;
+  private ExecutorService mFileSystemExecutorService;
+  private ExecutorService mUfsStateCacheExecutorService;
+  private MasterRegistry mRegistry;
+  private DefaultFileSystemMaster mFileSystemMaster;
+
+  @Before
+  public void before() throws Exception {
+    UserState us = UserState.Factory.create(Configuration.global());
+    AuthenticatedClientUser.set(us.getUser().getName());
+
+    mTempDir.create();
+    mUfsUri = mTempDir.newFolder().getAbsolutePath();
+
+    mUfs = new FlakyLocalUnderFileSystem(new AlluxioURI(mUfsUri),
+        UnderFileSystemConfiguration.defaults(Configuration.global()));
+    PowerMockito.mockStatic(UnderFileSystem.Factory.class);
+    Mockito.when(UnderFileSystem.Factory.create(anyString(), any())).thenReturn(mUfs);
+
+    Configuration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS);
+    Configuration.set(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, mUfsUri);
+    String journalFolderUri = mTempDir.newFolder().getAbsolutePath();
+
+    mFileSystemExecutorService = Executors
+        .newFixedThreadPool(4, ThreadFactoryUtils
+            .build("FileSystemMaster-%d", true));
+    mUfsStateCacheExecutorService = Executors
+        .newFixedThreadPool(4, ThreadFactoryUtils
+            .build("UfsStateCache-%d", true));
```

Comment:
strange indentation here
```suggestion
    mFileSystemExecutorService = Executors.newFixedThreadPool(4, 
        ThreadFactoryUtils.build("FileSystemMaster-%d", true));
    mUfsStateCacheExecutorService = Executors.newFixedThreadPool(
        4, ThreadFactoryUtils.build("UfsStateCache-%d", true));
```